README.md

```diff
@@ -56,7 +56,11 @@ module.exports = (config) => {
       // webpack watches dependencies
       // webpack configuration
     },
-
+    // optional middleware that blocks tests from running until code	      
+		// recompiles	      
+		beforeMiddleware: [	    
+			'webpackBlocker',
+		],
```

Comment:
is this even needed anymore? It seems to always wait for tests to recompile anyway